app/assets/javascripts/comment.js

```diff
@@ -51,3 +51,10 @@ function insertTitleSuggestionTemplate() {
   var newText = currentText+template;
   element.val(newText);
 }
+
+// JS API for submitting comments
+function addComment(comment, submitTo) {
```

Comment:
Great, and this works appropriately for nested comments (responding to other comments) as well as top-level comments? Just checking!